### PR TITLE
SUIT-10728 Resolve CodeClimate JS API issues.

### PR DESCRIPTION
**Pull request by JeKachan**
A lot of changes related to typescript tests. Splitting big functions in to smalls. Removed duplicated code where it make sense.
Also was skipped issues which fixed in Hotfix/suit-10520

### CODE REVIEW

File: lib/validataion/validators.js [hunk 1]
```diff
@@ -30,49 +30,58 @@ function throwError(text) {
...
+		.map(i => {
+			const dataName = getDataName(i);
+
+			if (dataName in ELEMENT_PROP_M) {
+				const elementPropTye = typeof dataName === 'string' ?
+					ELEMENT_PROP_TYPES[ELEMENT_PROP_M[dataName]] :
+					ELEMENT_PROP_TYPES[dataName];
+				const elementProp = getElementPropName(ELEMENT_PROP_M, dataName);
+
+				return `Element property '${elementProp}' should be of type ${elementPropTye}`;
```
Comment (AntonSBilan) [inline lib/validataion/validators.js]: `elementPropTye ` - typo in constant name and why not move it to `texts`

Comment (JeKachan) [reply lib/validataion/validators.js]: I dont noticed it, but I will fix it.

File: lib/utils/socketChainHelper.js [hunk 2]
```diff
@@ -120,9 +120,8 @@ const applyCountAndDelay = (socketMessage, data) => {
 /**
  * @description Apply default clicks array
  * @param {Object} socketMessage
- * @param {Object} data
  */
-const applyClicks = (socketMessage, data) => {
+const applyClicks = (socketMessage) => {
```
Comment (givehug) [inline lib/utils/socketChainHelper.js]: please remove redundant second argument in `applyCountAndDelayAndClicks`

File: lib/utils/socketErrorMessages.js [hunk 3]
```diff
@@ -95,6 +95,71 @@ function normalizeErrorType(response) {
 	return response.errorType;
 }
 
+function valTemplate(str, isActual) {
```
Comment (givehug) [inline lib/utils/socketErrorMessages.js]: Add some basic annotations please

File: lib/validataion/validators.js [hunk 4]
```diff
@@ -30,49 +30,58 @@ function throwError(text) {
 
 const getElementPropName = (mapOfNames, name) => typeof mapOfNames[name] === 'string' ? mapOfNames[name] : name;
 
+function prettifyElementPropsErrors(validate) {
```
Comment (givehug) [inline lib/validataion/validators.js]: please add some annotations

**Comment by andriichumak** [comment]
We'll merge it after Taras' console log refactoring.

**Comment by andriichumak** [comment]
@JeKachan please, resolve conflicts

**Comment by andriichumak** [comment]
@JeKachan conflicts again...

**Pull request merged**